samples/nrf9160/cloud_client/prj.conf

```diff
@@ -6,8 +6,11 @@
 
 # General config
 CONFIG_NEWLIB_LIBC=y
+CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=y
 CONFIG_RESET_ON_FATAL_ERROR=n
 CONFIG_NCS_SAMPLES_DEFAULTS=y
+CONFIG_LOG_MINIMAL=n
+CONFIG_LOG_IMMEDIATE=y
```

Comment:
Either use `NCS_SAMPLES_DEFAULT` or don't, but if you do don't override them :)
Why do we need `CONFIG_LOG_IMMEDIATE` here?

Reply:
Oops, this is some stuff I added during testing. Will remove both these lines